system/security/shared/caching.cpp

```diff
@@ -742,16 +757,22 @@ void CPermissionsCache::flush()
             delete (*ui).second;
         m_userCache.clear();
     }
+
     if (m_useLegacyDefaultFileScopePermissionCache)
     {
+        // MORE - is this safe? m_defaultPermission is unprotected,
```

Comment:
Should this code be here - or should it only be set when the cache is updated in the update thread?